### PR TITLE
Bug 1119515 - Implement initial search suggestions

### PR DESCRIPTION
**Pull request by daleharvey**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[daleharvey Dale  Harvey (daleharvey)](https://github.com/daleharvey) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=cfc8fe484791)

**Comment by try-server-hook** [comment]
[daleharvey Dale  Harvey (daleharvey)](https://github.com/daleharvey) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=033f97611fb6)

**Comment by try-server-hook** [comment]
[daleharvey Dale  Harvey (daleharvey)](https://github.com/daleharvey) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=6e655ddf8ef4)

**Comment by try-server-hook** [comment]
[daleharvey Dale  Harvey (daleharvey)](https://github.com/daleharvey) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=b44cf52c331c)

**Comment by try-server-hook** [comment]
[daleharvey Dale  Harvey (daleharvey)](https://github.com/daleharvey) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=5f9425dda337)

File: shared/test/integration/tbpl-manifest.json [hunk 1]
```diff
@@ -40,6 +40,11 @@
     "apps/system/test/marionette/software_home_file_open_error_test.js": "Bug 1119079 - Intermittent software_home_file_open_error_test.js | Software Home Button - File Open Error Proper layout for file error dialog",
     "apps/verticalhome/test/marionette/app_hosted_use_cached_icon_test.js": "Bug 1037194 - TEST-UNEXPECTED-FAIL | Vertical Home - Hosted app cached icon fetch fallback to default icon",
     "apps/verticalhome/test/marionette/collection_pin_result_uninstall_test.js": "Bug 1098310 - TEST-UNEXPECTED-FAIL | Vertical - Collection uninstall pinned collection web result",
-    "apps/verticalhome/test/marionette/collection_rename_test.js": "Bug 1090756 - TEST-UNEXPECTED-FAIL | Vertical - Collection Rename rename collection"
+    "apps/verticalhome/test/marionette/collection_rename_test.js": "Bug 1090756 - TEST-UNEXPECTED-FAIL | Vertical - Collection Rename rename collection",
+    "apps/system/test/marionette/rocketbar_activity_layout_test.js": "Bug 1117770 - Remove e.me",
```
Comment (KevinGrandon) [inline shared/test/integration/tbpl-manifest.json]: I think we should just remove these tests instead?

Comment (daleharvey) [inline shared/test/integration/tbpl-manifest.json]: Not all of them need to be removed, some do and some should be repurposed when the new functionality lands, or not depend on it, I think its best to disable then look at refactoring / repurposing them individually

Comment (KevinGrandon) [inline shared/test/integration/tbpl-manifest.json]: Which ones should be re-purposed? It seems like almost all of them are regarding E.me web results and bookmarking - do we plan on re-implementing any of that functionality?

Comment (daleharvey) [inline shared/test/integration/tbpl-manifest.json]: The search test will be applicable with the new functionality as will the browser_chrome_* ones. bookmark_web_result_test.js and activity_layout look likely to remove

**Pull request closed without merging**